texts/issues/messages.yml

```diff
@@ -1,63 +1,444 @@
+      url: https://wiki.eprints.org/w/OAI
+    -
+      # I have no idea how to make this
+      # but this should be there
+      caption: Mark resolved
+
+  details: [oai]
+
 -
   type: GENERIC_ATTACHMENT_URL
-  title: Generic document URL
+  matches: []
+  severity: WARNING
+  trigger: |
+    When the repository is misconfigured to use the default handle.net url
+  title: Unconfigured handle.net urls
+  description: |
+    The repository is exposing an unconfigured handle.net url. This means we
+    are unlikely to find full texts from your repository
+
+    {{ affectedUrl }}
+  resolution: |
+    Review the configuration for your handle.net persistent links
+  details: [affectedUrl]
+
+
+# Document Download
+# ==============================================================================
+#
+# Affects fulltext
+
 -
-  type: ROBOTS
-  title: Problems with the configured robots.txt
+  type: ROBOTS_URL_DISALLOWED
+  matches: [ROBOTS]
+  severity: ERROR
+  title: URL blocked by `robots.txt` rule
+  description: |
+    A rule in robots.txt prevents us from downloaded content from
+    {{ currentUrl }}
+  resolution: |
+    Revise your robots.txt rules in order to allow CORE to download content.
+  trigger: |
+    When we identify a rule in robots.txt which prevents us from downloading
+    from a url
+  details: [currentUrl]
+
+-
+  type: ROBOTS_MAXIMUM_CRAWL_DELAY_EXCEEDED
+  matches: []
+  severity: ERROR
+  title: Long crawl delay
+  description: |
+    In [`robots.txt` file]({{ link to robots }}),
+    a crawl delay has been set to {{ currentCrawlDelay }}.
+    It is longer than the maximum allowed rate of {{ maximumCrawlDelayLimit }}.
+  resolution: |
+    In order for CORE to harvest your repository in a reasonable amount of
+    time, we recommend lowering your crawl delay
+  trigger: |
+    If a crawl delay is longer than {{ maximumCrawlDelayLimit }}ms
+  details: [maximumCrawlDelayLimit, currentCrawlDelay]
+
 -
   type: ATTACHMENT_MALFORMED_URL
-  title: A document URL is malformed
+  severity: ERROR
+  trigger: |
+    When attempting a download in Document Download task, if a URL does not
+    conform to a standard URL, this issue is triggered.
+  title: URL is malformed
+  description: |
+    The URL {{ currentUrl }} is malformed.
+    {{#if sourceUrl }}It was discovered at {{ sourceUrl }}{{/if}}
+  resolution: |
+    Ensure that `dc:identifier` fields contain valid URLs and are correctly
+    URL encoded. For example, spaces should be replaced with `%20`. Further
+    details can be found at [should we write a guide or link elsewhere?]
+  details: [currentUrl, sourceUrl]
+
 -
   type: ATTACHMENT_NOT_VALID
-  title: A document is not valid
+  severity: ERROR
+  trigger: |
+    We check that a download is a pdf by checking the first 5 bytes contain
+    %pdf- If this is false, we set the PDF_NOT_VALID. In most cases, the issue
+    is replaced by a more specific issue as this issue is always triggered
+    when downloading HTML files.
```

Comment:
@samuelpearce what kind of HTML files we download?

Reply:
Usually when metadata provides a link to an HTML page such as [this page](https://aura.abdn.ac.uk/handle/2164/14314).

When we extract links from the HTML page and crawl for them, we check the downloaded file that it is a valid and well-formed pdf